tests/components/holiday/test_init.py

```diff
@@ -31,7 +29,9 @@ async def test_unload_entry(hass: HomeAssistant) -> None:
     await hass.config_entries.async_setup(entry.entry_id)
     await hass.async_block_till_done()
 
+    assert entry.state == ConfigEntryState.LOADED
+
     assert await hass.config_entries.async_unload(entry.entry_id)
     await hass.async_block_till_done()
 
-    assert entry.state is ConfigEntryState.NOT_LOADED
+    assert entry.state is not ConfigEntryState.LOADED
```

Comment:
Why did this change? It looked correct before.

Reply:
See above.